Add beta channel to /get-dart (#2362)
dart-lang/site-www: src/get-dart.md

</div>

## About release channels and version strings {#release-channels}

The Dart SDK has three release channels:

* **stable** channel: **stable releases**, updated roughly every three months;
  currently `[calculating]`{:.editor-build-rev-stable}.
  
  Stable releases are suitable for production use.
  
* **beta** channel: **preview releases**, usually updated every month;
  currently `[calculating]`{:.editor-build-rev-beta}.
  
  Beta channel builds are preview builds for the stable channel. We recommend
  testing, but not releasing, your apps against beta to preview new features, or
  test compatibility with future releases.
  
* **dev** channel: **pre-releases**, usually updated twice a week;
  currently `[calculating]`{:.editor-build-rev-dev}.
  
  Dev channel releases are the most current with latest changes, may be broken,
  are unsupported, and may contain unvetted breaking changes.

**Stable** channel releases of the Dart SDK have `x.y.z` version strings like
`1.24.3` and `2.1.0`. They consist of dot-separated integers, with no hyphens or
letters, where `x` is the major version, `y` is the minor version, and `z` is
the patch version.

> **Beta** and **Dev** channel releases of the Dart SDK (non-stable releases) have

Reviewer: ```suggestion
**Beta** and **dev** channel releases of the Dart SDK (non-stable releases) have
```

`x.y.z-a.b.<beta|dev>` versions like `2.8.0-20.11.beta`. The part before the

> hyphen follows the stable version scheme, `a` and `b` after the hyphen is the

Reviewer: ```suggestion
hyphen follows the stable version scheme, `a` and `b` after the hyphen are the
```

> pre-release and pre-release patch versions, and `beta` or `dev` is the channel.

Reviewer: ```suggestion
prerelease and prerelease patch versions, and `beta` or `dev` is the channel.
```

## About release channels and version strings

The Dart SDK has three release channels:

* **stable** channel: **stable releases**, updated roughly every three months;
  currently `[calculating]`{:.editor-build-rev-stable}.
  
  Stable releases are suitable for production use.
  
* **beta** channel: **candidate releases**, usually updated every month;
  currently `[calculating]`{:.editor-build-rev-beta}.
  

>   Beta channel builds are candidate builds for the stable channel. We recommend

Author: I'm just going to change "are candidate builds" to "are preview builds"
